cmd/background-controller/main.go

```diff
@@ -84,10 +85,12 @@ func main() {
 	var (
 		genWorkers      int
 		maxQueuedEvents int
+		emitEvents      string
 	)
 	flagset := flag.NewFlagSet("updaterequest-controller", flag.ExitOnError)
 	flagset.IntVar(&genWorkers, "genWorkers", 10, "Workers for the background controller.")
 	flagset.IntVar(&maxQueuedEvents, "maxQueuedEvents", 1000, "Maximum events to be queued.")
+	flagset.StringVar(&emitEvents, "emit-events", "", "Set this flag to a comma sperated list of PolicyViolation, PolicyApplied, PolicyError, PolicySkipped to disable events, e.g. --emit-events=PolicyApplied,PolicyViolation")
```

Comment:
Is this statement correct?

`emit-events` indicates one wants to generate that type of event, but the description says "to disable events".

Reply:
i think I should change it to omit events